compiler/modules/bir.sexpr/serialize.bal

```diff
@@ -97,15 +97,20 @@ function fromBasicBlock(FuncSerializeContext sc, bir:BasicBlock block, bir:File
 
 function formInsn(FuncSerializeContext sc, bir:Insn insn, bir:File file) returns Insn {
     if insn is bir:CallInsn {
-        bir:FunctionRef func =  insn.func;
+        (readonly & Operand)[] args = from var arg in insn.operands.slice(1) select fromOperand(sc, arg);
+        if insn is bir:CallIndirectInsn {
+            // JBUG: can't do
+            // "call"|"call-rest-list" insn_name = insn.resultArgIsList ? "call-rest-list" : "call";
+            // return [insn_name, fromRegister(sc, funcOperand), fromRegister(sc, insn.result), ...args];
```

Comment:
Not a JBUG. Spec improvement https://github.com/ballerina-platform/ballerina-spec/issues/1220

Reply:
Removed comment